distribution/lib/Standard/Base/0.0.0-dev/src/Warning.enso

```diff
@@ -188,3 +189,37 @@ merge_matched_warnings value matcher merger =
         new_warnings = merger (result.second.map .value)
         new_warnings.fold result.first acc-> warning->
             Warning.attach warning acc
+
+## PRIVATE
+
+type Prim_Warning
```

Comment:
```suggestion
## PRIVATE
type Prim_Warning
```

I'm not sure if such an empty newline at the end of the comment won't 'disconnect' the comment from the related piece of code? Regardless, we usually do not add such newlines.